src/demo/models/contracts/index.js

```diff
@@ -248,7 +249,7 @@ export const fields = {
     }
   },
   'parentContract': {
```

Comment:
make parentContract: {} because it has default values

Reply:
Done